pkg/cmd/server/api/v1/types.go

```diff
@@ -377,6 +377,9 @@ type AssetConfig struct {
 	// MetricsPublicURL is the public endpoint for metrics (optional)
 	MetricsPublicURL string `json:"metricsPublicURL"`
 
+	// CLIDownloadPublicURLs holds links to download client tools, where key is the platform name and value is the download URL (optional)
```

Comment:
Hmm, this seems super specific. Won't there be other things that need customizing on that page as well, like product name and other links? Plumbing all of that through config seems like the hard way to do that